src/woo_publications/metadata/tests/test_informatie_categorie_admin.py

```diff
@@ -0,0 +1,212 @@
+
+            self.assertEqual(response.status_int, 200)
+            # first item in list
+            self.assertInHTML(
+                "https://www.example.com/waardenlijsten/2",
+                str(response.content),
+            )
+            # second item not in list
+            with self.assertRaises(AssertionError):
+                self.assertInHTML(
+                    "https://www.example.com/waardenlijsten/1",
+                    str(response.content),
+                )
+
+        with self.subTest("filter_on_oorsprong"):
+            response = self.app.get(
+                url,
+                {"oorsprong__exact": InformatieCategorieOrigins.value_list},
+                user=self.user,
+            )
+
+            self.assertEqual(response.status_int, 200)
+            # first item in list
+            self.assertInHTML(
+                "https://www.example.com/waardenlijsten/1",
+                str(response.content),
+            )
+            # second item not in list
+            with self.assertRaises(AssertionError):
+                self.assertInHTML(
+                    "https://www.example.com/waardenlijsten/2",
+                    str(response.content),
+                )
+
+    def test_informatie_category_admin_can_not_update_item_with_oorsprong_waardenlijst(
+        self,
+    ):
+        url = reverse(
+            "admin:metadata_informatiecategorie_change",
+            kwargs={"object_id": self.information_category_value_list.id},
+        )
+
+        response = self.app.get(url, user=self.user)
+        form = response.forms["informatiecategorie_form"]
+
+        self.assertEqual(response.status_code, 200)
+        # page doesn't have save button.
+        with self.assertRaisesMessage(
+            AssertionError,
+            'Couldn\'t find \'<input class="default" name="_save" type="submit" value="Opslaan">\' in response',
+        ):
+            self.assertInHTML(
+                '<input type="submit" value="Opslaan" class="default" name="_save">',
+                str(response.content),
+            )
+
+        with self.assertRaisesMessage(
+            AssertionError,
+            "No field by the name 'identifier' found (fields: 'csrfmiddlewaretoken')",
+        ):
+            form["identifier"] = "https://www.example.com/waardenlijsten/12"
+
+    def test_informatie_category_admin_can_update_item_with_oorsprong_zelf_toegevoegd(
+        self,
+    ):
+        url = reverse(
+            "admin:metadata_informatiecategorie_change",
+            kwargs={"object_id": self.information_category_custom_entry.id},
+        )
+
+        response = self.app.get(url, user=self.user)
+        form = response.forms["informatiecategorie_form"]
+
+        self.assertEqual(response.status_code, 200)
+        # page has save button.
+        self.assertInHTML(
+            '<input type="submit" value="Opslaan" class="default" name="_save">',
+            str(response.content),
+        )
```

Comment:
same remarks as earlier - use webtest for the actual form submitting and check what the result is of that

basically when using webtest, you're not supposed to be dealing with (raw) HTML yourself in your tests.